### PR TITLE
organizing stylesheets and scripts

### PR DESCRIPTION
**Pull request by iris-beckham**
- css files are in stylesheets folder

- js files are in scripts folder

- top navigation bar loads through javascript to each html page

- functional hamburger drawer

### CODE REVIEW

File: website/src/main/webapp/stylesheets/style.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  size: 24px;
+  letter-spacing: 0px;
+}
+
+h6{
+  font: Roboto;
+  font-weight: medium;
+  size: 20px;
+  letter-spacing: 0.15px;
+}
```
Comment (hajiler) [inline website/src/main/webapp/stylesheets/style.css]: Nit: Add new line at end of file

**Review by hajiler — APPROVED** [review]
LGTM

**Review by tonypan2000 — CHANGES_REQUESTED** [review]
Please fix Javascript errors and merge conflicts

**Comment by tonypan2000** [comment]
Issues:
1. search button on nav bar on index.html throws javascript error
2. Hamburger dropdown covers website name
3. Hamburger does not work on organization.html
4. recommend.html does not work, nav bar does not get added

**Pull request merged**